### PR TITLE
Automated cherry pick of #2142: Default LogCollector instance first-thing

### PR DESCRIPTION
**Pull request by rene-dekker**
Cherry pick of #2142 on release-v1.28.

#2142: Default LogCollector instance first-thing

### CODE REVIEW

**Comment by marvin-tigera** [comment]
OK, I will merge the pull request when it's ready, leave the commits as is when I merge it, and leave the branch after I've merged it.

**Review by caseydavenport — APPROVED** [review]

**Pull request merged**
